controllers/csi/provisioner/reconciler.go

```diff
@@ -34,22 +34,27 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/record"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-var (
-	log = logger.NewDTLogger().WithName("provisioner")
+const (
+	failedInstallAgentVersionEvent = "FailedInstallAgentVersion"
+	installAgentVersionEvent       = "InstallAgentVersion"
```

Comment:
The _Version_ component in the name and in the description is pretty redundant. Installing an agent is always in context of a version.